graph_adventure/adv.py

```diff
@@ -9,37 +9,98 @@
 
 world.loadGraph(roomGraph)
-world.printRooms()
+# world.printRooms()
 player = Player("Name", world.startingRoom)
 
 
-# FILL THIS IN
-traversalPath = ['n', 's']
+# def dft(self, starting_vertex, traversalGraph):
+#     if traversalGraph is None:
+#         visited = [False] * (len(self.vertices) + 1)
+#
+#     traversalGraph[starting_vertex] = True
+#
+#     for i in self.vertices[starting_vertex]:
+#         if traversalGraph[i] is False:
+#             self.dft(i, traversalGraph)
+#
+#     return traversalGraph
 
 
 # TRAVERSAL TEST
-visited_rooms = set()
+visited_rooms = {}
+
+# traversalPath = dft(traversalGraph)
+traversal_path = []
+
 player.currentRoom = world.startingRoom
-visited_rooms.add(player.currentRoom)
-for move in traversalPath:
+
+def establish_exits (currentRoom):
```

Comment:
Great job, converting the instructions/spec to code Kyle. Method names are very clear. I especially like this method `establish_exits` because the code tracks so closely with the description in the repo 👍